website/src/css/customTheme.scss

```diff
@@ -1865,3 +1865,13 @@ html[data-theme="light"].blog-wrapper {
     }
   }
 }
+
+/*
+Prevent useless bottom margin for multiline <figcaption> tags in MDX:
+<figcaption>
+  some paragraph text
+</figcaption>
+ */
+figcaption > p:last-child {
+  margin-bottom: 0;
+}
```

Comment:
This needs a new line at the end of the file it seems

Reply:
I added a new line but not sure why it's needed? CI is passing. Is there linter I don't see?

Reply:
I mentioned it because GitHub showed this icon presumably because there wasn't an extra new line, whereas all other files have that extra new line. 
![CleanShot 2023-09-15 at 17 48 25](https://github.com/facebook/react-native-website/assets/249164/a7c17bd1-e021-45c1-85be-675faa24cb56)

And yeah, it's normally enforced with a linter. See react-native's main eslint config: https://github.com/facebook/react-native/blob/2be409ff55b50563a5f123f7823fa7cdb72dbef9/packages/eslint-config-react-native/index.js#L264